src/common/src/types/interval.rs

```diff
@@ -283,6 +283,14 @@ impl IntervalUnit {
     pub fn is_positive(&self) -> bool {
         self > &Self::new(0, 0, 0)
     }
+
+    pub fn min() -> Self {
+        Self {
+            months: i32::MIN,
+            days: i32::MIN,
+            ms: i64::MIN,
+        }
+    }
```

Comment:
What's the use of a `min` interval?

Reply:
Idk for now. I just left it there for potential usage.

Reply:
Okay. Just a bit weird here because the semantic seems opposite to watermark 🤣 But let's just keep this for now.